Assets/PlayerAssets/Scripts/Inventory.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using ARPGTD.CharacterStats;
+using UnityEngine;
+
+[System.Serializable]
+public class Inventory : MonoBehaviour {
+
+	public List<Item> inventory;
+
+	// Use this for initialization
+	void Start () {
+		inventory = new List<Item>();
+		for(int i=0;i<6;i++){
+			inventory.Add(null);
+		}
+	}
+	
+	// Update is called once per frame
+	void Update () {
+		
+	}
+
+	public void AddItemToInventory(Item item){
+        for(int i=0; i<6; i++){
+            if(inventory[i] == null){
+                inventory[i] = item;
+                break;
+            }
+        }
+        calcStats(item);
+    }
+
+    public void RemoveItemFromInventory(Item item){
+        for(int i=0; i<6; i++){
+            if(inventory[i] == item){
+                inventory[i] = null;
+                break;
+            }
+        }
+        calcStats(item);
+    }
+
+    public Item GetItemInInventory(int itemPos){
+        return inventory[itemPos];
+    }
+
+    public void calcStats(Item item){
+        List<int> stats = new List<int>(); 
+        for(int i=0; i<6; i++){
```

Comment:
this '6' is appearing everywhere. I'm guessing it's the inventory slots but that's not obvious, make it a variable